components/Dashboard/DashboardWrapper.tsx

```diff
@@ -43,38 +44,50 @@ interface Props {
   children: React.ReactChild;
 }
 
-const DashboardWrapper = ({ children }: Props): React.ReactElement => (
-  <>
-    <Head>
-      <title>Dashboard | Social care | Hackney Council</title>
-    </Head>
+const DashboardWrapper = ({ children }: Props): React.ReactElement => {
+  const { getConfigValue } = useAppConfig();
 
-    <div className={`govuk-grid-row ${s.outer}`}>
-      <div className="govuk-grid-column-one-quarter">
-        <div className={s.sticky}>
-          <nav>
-            <ul className="lbh-list">
-              {navigation.map((link) => (
-                <NavLink href={link.href} key={link.href}>
-                  {link.text}
-                </NavLink>
-              ))}
+  let navLink;
 
-              <ConditionalFeature name="workflows-pilot">
-                <NavLink
-                  href={process.env.NEXT_PUBLIC_WORKFLOWS_PILOT_URL as string}
-                >
-                  Workflows
-                </NavLink>
-              </ConditionalFeature>
-            </ul>
-          </nav>
-          <MyData />
+  try {
+    navLink = (
+      <NavLink href={getConfigValue('workflowsPilotUrl') as string}>
```

Comment:
Instead of casting here, we could make `getConfigValue` take a restricting generic type argument, something like:

```tsx
getConfigValue<string>("workflowsPilotUrl");
```

Happy to pair with you to go over how to do this

Reply:
We decided it's fine as is as we don't gain too much from it in this case.